File: 1115.cpp
#include <iostream>
using namespace std;
int main(){
    while(true){
        int a, b;
        cin >> a >> b;
        if(a > 0 and b > 0){
            cout << "primeiro" << endl;
        }
        if(a < 0 and b > 0){
            cout << "segundo" << endl;
        }
        if(a < 0 and b < 0){
            cout << "terceiro" << endl;
        }
        if(a > 0 and b < 0){
            cout << "quarto" << endl;
        }
        if(a == 0 or b == 0){
            break;
        }
    }
}

File: 1115.java
import java.util.Scanner;
public class Main {
public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        while(true){
                int a = input.nextInt(); int b = input.nextInt();
                if(a == 0 || b == 0){
                        break;
                }
                if(a > 0) {
                        System.out.println((b > 0)?"primeiro":"quarto");
                }
                else{
                        System.out.println((b > 0)?"segundo":"terceiro");
                }
        }
}
}
